metricbeat/module/elasticsearch/ccr/_meta/data.json

```diff
@@ -1,39 +0,0 @@
-{
```

Comment:
@sayden Looks like this file got deleted. Any chance you could regenerate it? The other Stack Monitoring PRs have it so it would be good to have it in this PR too.

Reply:
Okay. I have added a `TestData` function to work with mock input data and now we have a `data.json` :tada: